src/cloud/docker/docker-containers.md

```diff
@@ -65,52 +66,50 @@ Docker volumes are used to maintain data throughout the lifecycle of the Docker
 
 You do not interact with most of these volumes, which are used by the Docker containers and follow the docker-compose lifecycle. The only exception to this is the `magento-sync` directory which is an external volume used by the Mutagen application to transport data into the containers from the host operating system.
 
-You can remove all data, and rebuild a clean environment using the following command:
-```
+### Rebuild a clean environment
+
+Use the following command to remove all data and rebuild a clean environment:
+
+```bash
  bin/docker down
- # this triggers a docker-compose command which removes volumes
- # docker-compose down -v
 ```
-If you are using Mutagen for file synchronization, this command does not remove the `magento-sync` volume, which is an external volume that you have to create or delete manually.
+
+This command runs the `docker-compose down -v` command which removes all components of your local Docker instance including containers, networks, volumes, and images. This command does not [the persistent database volume][db] or the `magento-sync` volume used for file synchronization.
+
+The `magento-sync` volume is an external volume that you have to create or delete manually.
```

Comment:
```suggestion
```

Reply:
May be better to use "must" or something. "have to" is a bit clunky.